app/models/item.js

```diff
@@ -41,6 +46,6 @@ export default Model.extend({
     }),
     scheduleFilterText: Ember.computed('title', 'location', 'startTimeFormatted', function () {
         return this.get('title') +
-            this.get('location') + this.get('startTimeFormatted') + this.get('userName');
```

Comment:
multi-line code should be indented [link](https://contribute.jquery.org/style-guide/js/#multi-line-statements)